script.js

```diff
@@ -0,0 +1,34 @@
+var app = new Vue({
+  el: "#app",
+  data: {
+    writedTask: "",
+    TodoId: 1,
+    items: [],
+  },
+  methods: {
+    addNewTask: function () {
+      if (this.writedTask === "") {
+        return items;
+      } else {
+        this.items.push({
+          id: this.TodoId++,
```

Comment:
There is an issue in how you handled `id`s here.

If we add two tasks:
![image](https://user-images.githubusercontent.com/42524280/185463526-4ec80f1a-c63f-43f6-884e-49fa392327f0.png)

Then delete the first one:
![image](https://user-images.githubusercontent.com/42524280/185463581-666eed72-6f0b-487b-a469-82ab40f4d8ea.png)

Then add it again, it will have the id of `3`:
![image](https://user-images.githubusercontent.com/42524280/185463629-269ac88a-55ca-4cdc-be47-7077f8075ef3.png)